reflex-dom-core/src/Reflex/Dom/Builder/Immediate.hs

```diff
@@ -144,8 +150,8 @@ import Data.String (IsString)
 import Data.Text (Text)
 import Foreign.JavaScript.Internal.Utils
 import Foreign.JavaScript.TH
-import GHCJS.DOM.Document (Document, createDocumentFragment, createElement, createElementNS, createTextNode, createComment)
-import GHCJS.DOM.Element (getScrollTop, removeAttribute, removeAttributeNS, setAttribute, setAttributeNS, hasAttribute)
+import "ghcjs-dom" GHCJS.DOM.Document (Document, createDocumentFragment, createElement, createElementNS, createTextNode, createComment)
```

Comment:
Is PackageImports really necessary here?

Reply:
No, accidentally added this when committing. This was because `ob run` gets confused here.